src/components/MapControls/MapControls.js

```diff
@@ -48,15 +48,31 @@ function MapControls({onChange, mapDataset}) {
     }, []);
     
     // this useEffect responds to changes in the selected dataset, via the mapDataset
-    // prop. It fetches the minimum and maximum values of the dataset and stores them.
-    // It is inefficient - the dataset minimum and maximum do not change if
-    // a user merely goes to the next timestamp, but they will be fetched anyway.
-    // this is such a small request it is not an issue.
+    // prop. It fetches the minimum and maximum values of the dataset.
+    // It uses the variable configuration file in the variable config yaml, if possible.
+    // If the config is unavailable or doesn't have an entry for the variable,
+    // it requests dataset minmax from ncWMS, which is frequently inaccurate, as a
+    // fallback. We suspect that ncWMS either returns the minmax of only the first
+    // timestamp, which tends to result in, for example, water temperature datasets that
+    // look good in January but are completely maxed out in June.
+    // This function is inefficient, as the dataset minmax is not changed if the
+    // user merely goes to a different timestamp, but it's fast enough we don't
+    // currently care.
     useEffect(() => {
```

Comment:
Given the shortcomings (and unknown factors: "We suspect ...") of ncWMS minmax, it would probably be better simply to impose the condition that every variable specify a min and max. We're in control of all of that, so it's a matter only of diligence (and helpful error messages and some kind of hardcoded -- or config coded -- fallback).

Such a change would also greatly simplify this code: We would not need a `useEffect` (because not potential async call to ncWMS), and would not need manage `minMax` as a state variable. Just extract the min/max from the config and be done with it. See somewhat superseded remark below for a bit more on that.